### PR TITLE
Changed seeders

### PR DESCRIPTION
**Pull request by james-martinez0**
I splited seeder based on module

Create your own seeder codes

I added accounts for each user type

I also added new user account with my plm email to test the login with microsoft account

Please run 'php artisan:fresh --seed' to test your code. So that all dev have the same data at the database

### CODE REVIEW

File: database/migrations/2023_11_09_122608_create_students_table.php
Comment (jhaimec) [inline database/migrations/2023_11_09_122608_create_students_table.php:1]: nasa `users` table na yung `student_id`, yun yung `school_no`. lahat nmn ng users may school no mapa employee or student kaya ganun ginawa ko

File: database/migrations/2023_11_09_123531_create_rooms_table.php
Comment (jhaimec) [inline database/migrations/2023_11_09_123531_create_rooms_table.php:1]: `buildings_id` dapat sya with "s".

File: database/seeders/AdminSeeder.php
Comment (jhaimec) [inline database/seeders/AdminSeeder.php:1]: dito ginawa ko ng string yung `code` purpose kasi nun maging abbreviation ng name/title like "Gusaling Ejercito" magiging "GE" or "GEE", mas okay kaysa "002" diba

File: database/migrations/2023_11_09_123531_create_rooms_table.php
Comment (james-martinez0) [reply database/migrations/2023_11_09_123531_create_rooms_table.php:1]: alam ko automatic naglalagay ng 's' si laravel like yung sa ibang table user_id sya imbes na users_id. ayan ata dapat yung syntax ng relationship

**Review by jhaimec — COMMENTED** [review]
pacheck nlng comments ko

File: database/seeders/AdminSeeder.php
Comment (james-martinez0) [reply database/seeders/AdminSeeder.php:1]: ok nabago ko na seeder

**Comment by james-martinez0** [comment]
sige. di ko napansin yung branch mo, sa main at kay Tom lang ako tumingin. merge ko din yung sayo.

File: database/migrations/2023_11_09_123531_create_rooms_table.php
Comment (jhaimec) [reply database/migrations/2023_11_09_123531_create_rooms_table.php:1]: wait test ko. last time kasi walang lumabas na option na foreign keys sa phpmyadmin ee, lumabas lng ng nilagyan ko ng "s".

File: database/migrations/2023_11_09_122608_create_students_table.php
Comment (james-martinez0) [reply database/migrations/2023_11_09_122608_create_students_table.php:1]: done

**Comment by james-martinez0** [comment]
Done na po. except dun sa buidling_id

File: database/migrations/2023_11_09_123531_create_rooms_table.php
Comment (jhaimec) [reply database/migrations/2023_11_09_123531_create_rooms_table.php:1]: keep yung `building_id`

**Comment by jhaimec** [comment]
sa seeders susundin yung gawa ni martinez

**Review by jhaimec — APPROVED** [review]

**Pull request merged**